### PR TITLE
Add documentation for DROP integration

### PR DESCRIPTION
**Pull request by pfrazer**
## Proposed change

Add documentation for a new DROP integration. Information about the product line using this new integration is at https://dropconnect.com/ .

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/104319
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4908
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/drop_connect.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- **Salt Sensor**: alerts when the salt level in the softener brine tank is low.
+
+### Configuration
+
+To use DROP with Home Assistant, you must have already configured the [MQTT](/integrations/mqtt/) platform.
+
+To enable the DROP integration, use the DROP Connect app to connect your DROP Hub to your MQTT broker:
+- Start the DROP Connect app and connect to your DROP Hub.
+- In the 'Hub Network Settings' section on the System > Advanced page, tap the 'Configure MQTT' button.
+- Enter the MQTT broker address, port number, username, and password.
```
Comment (jbouwh) [inline source/_integrations/drop_connect.markdown:54]: May be you could explain on how to configure a username and password on the Mosquitto Addon as the default username and password on the MQTT broker are set up automatically and are not known by the user.

afbeelding

Comment (pfrazer) [reply source/_integrations/drop_connect.markdown:54]: Additional instructions added in c5bbd4bdfa87654a60b0854d14adeaf3eb80a2e9.

**Review by jbouwh — APPROVED** [review]
LGTM,
Thnx @pfrazer 👍

File: source/_integrations/drop_connect.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+To enable the DROP integration, use the DROP Connect app to connect your DROP Hub to your MQTT broker:
+
+- Start the DROP Connect app and connect to your DROP Hub.
+- In the **Hub Network Settings** section on the **System > Advanced** page, tap the **Configure MQTT** button.
+- Enter the MQTT broker address, port number, username, and password.
+- Click **Connect** and confirm that the DROP Hub has connected to the MQTT broker.
+
+Once the DROP Hub is connected to your MQTT broker, the devices on your DROP system will be discovered by Home
+Assistant.
+Go to **Settings > Devices & Services** in the Home Assistant UI to add your DROP devices.
```
Comment (frenck) [inline source/_integrations/drop_connect.markdown:59]: This isn't the correct way to guide users to set up an integration. We have templates for that available, which must be used.

@pfrazer Please address this in a new PR.

../Frenck

File: source/_integrations/drop_connect.markdown [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- **DROP Hub**: centralized system status and control.
+- **Leak Detector**: reports detected leaks around the home.
+- **Softener**: improves water quality by decreasing water hardness with integral water shutoff.
+- **Filter**: improves water quality by removing contaminants with integral water shutoff.
+- **Protection Valve**: automatic water shutoff with slow leak detection.
+- **Pump Controller**: smart replacement for an FSG pressure switch.
+- **RO Filter**: reverse osmosis drinking water filtration.
+- **Salt Sensor**: alerts when the salt level in the softener brine tank is low.
+
+### Configuration
```
Comment (frenck) [inline source/_integrations/drop_connect.markdown:35]: This is not integration configuration, but device configuration.

This section should be called "Pre-requisites" instead.

Please address this in a new PR

File: source/_integrations/drop_connect.markdown [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+Under **Configuration**, enter a new username and password for your DROP Hub in the **Logins** field:
+
+```yaml
+- username: drop
+  password: your_unique_password
+```
+
+If you are using a different MQTT broker, please reference its documentation to learn how to add a new user.
+
+To enable the DROP integration, use the DROP Connect app to connect your DROP Hub to your MQTT broker:
```
Comment (frenck) [inline source/_integrations/drop_connect.markdown:50]: We have many users not using add-ons... this is super confusing as this is integration documentation and not device or add-on documentation

**Review by frenck — COMMENTED** [review]
@pfrazer Please take a look at these comments and address those in a new PR.

**Comment by frenck** [comment]
/CC @jbouwh, please mark PRs as these as `parent-merged` so the docs team can take a look before merging. Thanks 👍

File: source/_integrations/drop_connect.markdown (hunk 4 above)
Comment (jbouwh) [reply source/_integrations/drop_connect.markdown:50]: I was the one who suggested this, I'll open a followup PR on next.

Comment (jbouwh) [reply source/_integrations/drop_connect.markdown:50]: https://github.com/home-assistant/home-assistant.io/pull/30527

File: source/_integrations/drop_connect.markdown (hunk 3 above)
Comment (jbouwh) [reply source/_integrations/drop_connect.markdown:35]: Address in #30527

File: source/_integrations/drop_connect.markdown (hunk 2 above)
Comment (jbouwh) [reply source/_integrations/drop_connect.markdown:59]: Addressed in #30527

**Pull request merged**